app/assets/stylesheets/custom.css.scss

```diff
@@ -44,6 +52,45 @@ body {
   }
 }
 
+.smfont {
+  font-size: 0.9em;
+}
+
+.warning {
+  color: $red;
+  padding-left: 30px;
+}
+
+.card {
+  background-color: $white;
+  border: 1px solid transparent;
+  border-color: #BCE8F1;

```

Comment:
Color literals like `#BCE8F1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#BCE8F1` should be written as `#bce8f1`